### ISSUE TITLE
Support: "Independent support layer height" active although hidden

### ISSUE THREAD

**Issue by avogra** [SoftFever/OrcaSlicer#5668]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### OrcaSlicer Version

2.0.0
2.1.1

### Operating System (OS)

Windows

### OS Version

Windows 11

### Printer

BambuLab X1C

### How to reproduce

1. Enable Support with type normal(auto) and disable Others->Prime Tower
2. enable Support -> Advanced -> Independent support layer height
3. enable Others -> Prime Tower

### Actual results

after step 2., in addition to the part layers, Orca introduces layers at some unexpected layer heights
after step 3., the Option "Independent support layer height" is hidden but supports are still in separate layers with odd heights

### Expected results

after 3. odd layers don't appear and support is generated at the same layers as the part.
In other words: disable "Independent support layer height" instead of only hiding it, when prime tower is enabled.

as I learned from https://github.com/SoftFever/OrcaSlicer/issues/2806, hiding of the option "Independent support layer height" is intended, because it is not compatible with prime tower. Yet, activating prime tower while the option is enabled seems to only hide the option but not deactivated it.
It is at least irritating for the user to see those odd layers with no apparent way to get rid of them. 
If prime tower is not compatible with "independent support layer height", then it should be disabled in the first place, not only hidden.

### Project file & Debug log uploads

[support_layer_height_bug.zip](https://github.com/user-attachments/files/15793644/support_layer_height_bug.zip)

### Checklist of files to include

- [ ] Log file
- [X] Project file

### Anything else?

side note: tree support is always generated on the same layers as the parts, no matter if the option is enabled. so if the option doesn't work with tree support, it should be hidden there. Or if it does work, then it isn't applied correctly.

_No response_

**Comment by avogra** [comment]
Issue is still open.

**Comment by avogra** [comment]
Tried it again in 2.1.1. It is still present. This should be a very easy fix, that avoids unexpected behaviour for some people

**Comment by CCS86** [comment]
Agreed.

**Comment by CCS86** [comment]
Bump

**Comment by ChevaTomme** [comment]
I agree, this a confusing bug, i was looking for a way to control support layer height and did not know about this bug hiding the setting until i found this post.

**Comment by CCS86** [comment]
Bump

**Comment by valerii-bokhan** [comment]
It will be somewhat fixed in https://github.com/SoftFever/OrcaSlicer/pull/10697

But it's better to decide what to do with this option. I’d remove it completely, or at least set it to `false` by default.
